app/google-services (1).json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
I think you want to rename this file

Reply:
Didn't notice it kept it. All fixed now.